cime_config/tests.py

```diff
@@ -822,10 +822,11 @@
     "e3sm_wav_developer" : {
         "time"    : "0:45:00",
```

Comment:
We should probably up this to `1:00:00` to prevent timeout issues with running the new B-case+waves tests

Reply:
is 1:00:00 enough? or do you think we need more than that?